### PR TITLE
runner/1 Basic Cairo Verifier Runner Setup

### PR DESCRIPTION
**Pull request by neotheprogramist**

### CODE REVIEW

File: README.md
Comment (tiagofneto) [inline README.md:1]: Maybe let's move this README to the runner directory

File: runner/README.md
Comment (tiagofneto) [inline runner/README.md:1]: This README makes more sense in the root of the repo perhaps?

File: runner/resources/fulloutput.txt
Comment (tiagofneto) [inline runner/resources/fulloutput.txt:1]: Maybe we should include this resources files in the `.gitignore`? Or are they supposed to serve as examples here?

Seems annoying that every time we will parse new proofs we will have to commit them

File: runner/resources/parserin.txt
Comment (tiagofneto) [inline runner/resources/parserin.txt:1]: Same comment as the `fulloutput.txt` file

File: src/lib.cairo [hunk 1]
```diff
@@ -1,3 +1,10 @@
 mod channel;
 mod common;
+mod input_structs;
 mod structs;
+
+use cairo_verifier::input_structs::stark_proof::StarkProof;
+
+fn main(stark_proof: StarkProof) -> (felt252, felt252) {
```
Comment (tiagofneto) [inline src/lib.cairo:8]: I think for now it's ok to leave the main function in the main `lib` file as it is quite simple, but if it starts getting more complex we should move it to a separate file

**Review by tiagofneto — CHANGES_REQUESTED** [review]

File: runner/resources/fulloutput.txt
Comment (neotheprogramist) [reply runner/resources/fulloutput.txt:1]: Sure I agree, the goal of it was to show you the format of the proof files - just to present them

**Review by tiagofneto — APPROVED** [review]

**Pull request merged**